gift-certificate-application/controller/src/main/java/com/epam/esm/controller/Application.java

```diff
@@ -0,0 +1,15 @@
+package com.epam.esm.controller;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
+
+@SpringBootApplication(scanBasePackages = "com.epam.esm")
+//@ComponentScan("com.epam.esm")
```

Comment:
Remove this commented line, such lines should never be in commits.

Reply:
I have made it.